src/ess/livedata/dashboard/reduction.py

```diff
@@ -53,6 +56,23 @@ def __init__(
         )
         self._logger.info("Reduction dashboard initialized")
 
+    def _create_announcements_pane(self) -> pn.pane.Markdown:
+        """Create a Markdown pane that periodically reloads from file."""
+
+        def read_announcements() -> str:
+            try:
+                return ANNOUNCEMENTS_FILE.read_text()
+            except FileNotFoundError:
+                return "*No announcements file found.*"
+
+        pane = pn.pane.Markdown(read_announcements(), sizing_mode='stretch_width')
+
+        def refresh():
+            pane.object = read_announcements()
+
+        pn.state.add_periodic_callback(refresh, period=30_000)  # 30 seconds
+        return pane
```

Comment:
Shouldn't this be fetching from a remote url instead of reading from disk?

We shouldn't redeploy the whole application just to update annoucements.